src/Commands/Auth/WhoamiCommand.php

```diff
@@ -17,8 +25,23 @@ class WhoamiCommand extends TerminusCommand
      *   Responds with the email of the logged-in user
      * @usage terminus auth:whoami -vvv
      *   Responds with the current session and user's data
+     * @field-labels
+     *   firstname: First Name
+     *   lastname: Last Name
+     *   email: Email
+     *   id: ID
+     *
+     * @return string
      */
-    public function whoami() {
+    public function whoAmI() {
+        $auth = new Auth();
+        if ($auth->loggedIn()) {
+            $user = Session::getUser()->fetch()->serialize();
+            return ResultData::message(new RowsOfFields([$user,]))->getMessage();
```

Comment:
Reading this code a bit more in context, you should be able to just return the RowsOfFields object directly.